Stackl create instance fails when policies not defined

# Expected Result
 
Stackl should ignore policies when it is not filled in in the stack instance. 

# Actual Result

Stackl gives no error when applying a document with `policies` key, but does error when trying to create a stack instance from a sat containing no policies key.

Core logs:
```
[2020-10-19 11:25:00 +0000] [8] [ERROR] Exception in ASGI application
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/uvicorn/protocols/http/httptools_impl.py", line 390, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/middleware/proxy_headers.py", line 45, in __call__
    return await self.app(scope, receive, send)
  File "/usr/local/lib/python3.8/site-packages/fastapi/applications.py", line 146, in __call__
    await super().__call__(scope, receive, send)
  File "/usr/local/lib/python3.8/site-packages/starlette/applications.py", line 102, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/usr/local/lib/python3.8/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc from None
  File "/usr/local/lib/python3.8/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/usr/local/lib/python3.8/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc from None
  File "/usr/local/lib/python3.8/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.8/site-packages/starlette/routing.py", line 550, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.8/site-packages/starlette/routing.py", line 227, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.8/site-packages/starlette/routing.py", line 41, in app
    response = await func(request)
  File "/usr/local/lib/python3.8/site-packages/fastapi/routing.py", line 196, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.8/site-packages/fastapi/routing.py", line 148, in run_endpoint_function
    return await dependant.call(**values)
  File "/app/core/routers/stack_instances_router.py", line 97, in post_stack_instance
    (stack_instance, return_result) = stack_manager.process_stack_request(
  File "/app/core/manager/stack_manager.py", line 50, in process_stack_request
    stack_instance, err_message = handler.handle(job)
  File "/app/core/handler/stack_handler.py", line 26, in handle
    return self._handle_create(item['document'])
  File "/app/core/handler/stack_handler.py", line 279, in _handle_create
    for policy_name, attributes in stack_app_template.policies.items():
AttributeError: 'NoneType' object has no attribute 'items'
```

# Steps To Reproduce

Use following docs for SAT:

```
category: configs
description: example-playbook
invocation:
  generic:
    description: Example playbook
    image: stacklio/example-playbook:v1.0.0
    tool: ansible
name: example-playbook
params:
  ansible_playbook_path: /opt/ansible/playbook.yml
type: functional_requirement
```

```
category: configs
description: example-playbook
name: example-playbook
services:
- example-playbook
type: stack_application_template
```

```
category: configs
description: example-playbook
name: example-playbook
services:
- example-playbook
type: stack_application_template
```

Use following commands:

```
stackl apply -d .
stackl create instance --stack-application-template example-playbook --stack-infrastructure-template sit instance-name
```

# Potential Causes, Fixes or Additional Information

Don't loop over SAT policies when not defined. See https://github.com/stacklio/stackl/blob/master/stackl/core/core/handler/stack_handler.py#L255

--- a/stackl/core/core/handler/stack_handler.py
+++ b/stackl/core/core/handler/stack_handler.py
@@ -330,23 +330,24 @@ class StackHandler(Handler):
 
         opa_service_params = tree()
         # Verify the SAT policies
-        for policy_name, attributes in stack_app_template.policies.items():
-            policy = self.document_manager.get_policy_template(policy_name)
-            for policy_params in attributes:
-                new_result = self.evaluate_sat_policy(policy_params, opa_data,
-                                                      policy, item.params, item.replicas)
+        if stack_app_template.policies:
+            for policy_name, attributes in stack_app_template.policies.items():
+                policy = self.document_manager.get_policy_template(policy_name)
+                for policy_params in attributes:
+                    new_result = self.evaluate_sat_policy(policy_params, opa_data,
+                                                          policy, item.params, item.replicas)
 
-                if not new_result['fulfilled']:
-                    logger.error(
-                        f"[StackHandler]: Opa result message: {new_result['msg']}"
-                    )
-                    return None, new_result['msg']
+                    if not new_result['fulfilled']:
+                        logger.error(
+                            f"[StackHandler]: Opa result message: {new_result['msg']}"
+                        )
+                        return None, new_result['msg']
 
-                process_service_targets(policy_params,
-                                        new_result,
-                                        service_targets,
-                                        opa_service_params,
-                                        outputs=policy.outputs)
+                    process_service_targets(policy_params,
+                                            new_result,
+                                            service_targets,
+                                            opa_service_params,
+                                            outputs=policy.outputs)
 
         service_targets = self.evaluate_replica_policy(item, service_targets)
         if not service_targets['result']['fulfilled']:
@@ -595,28 +596,25 @@ class StackHandler(Handler):
 
         opa_service_params = tree()
         # Verify the SAT policies
-        for policy_name, attributes in stack_application_template.policies.items():
-            policy = self.document_manager.get_policy_template(policy_name)
-            for policy_params in attributes:
-                new_result = self.evaluate_sat_policy(policy_params, opa_data,
-                                                      policy,
-                                                      {
-                                                          **stack_instance.instance_params,
-                                                          **item.params
-                                                      },
-                                                      item.replicas)
+        if stack_application_template.policies:
+            for policy_name, attributes in stack_application_template.policies.items():
+                policy = self.document_manager.get_policy_template(policy_name)
+                for policy_params in attributes:
+                    new_result = self.evaluate_sat_policy(policy_params, opa_data,
+                                                          policy, {**stack_instance.instance_params, **item.params},
+                                                          item.replicas)
 
-                if not new_result['fulfilled']:
-                    logger.error(
-                        f"[StackHandler]: Opa result message: {new_result['msg']}"
-                    )
-                    return None, new_result['msg']
+                    if not new_result['fulfilled']:
+                        logger.error(
+                            f"[StackHandler]: Opa result message: {new_result['msg']}"
+                        )
+                        return None, new_result['msg']
 
-                process_service_targets(policy_params,
-                                        new_result,
-                                        service_targets,
-                                        opa_service_params,
-                                        outputs=policy.outputs)
+                    process_service_targets(policy_params,
+                                            new_result,
+                                            service_targets,
+                                            opa_service_params,
+                                            outputs=policy.outputs)
         if item.replicas != {}:
             service_targets = self.evaluate_replica_policy(item, service_targets)
         else:

